pkg/deploy/api/test/ok.go

```diff
@@ -240,7 +240,7 @@ func OkHPAForDeploymentConfig(config *deployapi.DeploymentConfig, min, max int)
 		Spec: autoscaling.HorizontalPodAutoscalerSpec{
 			ScaleTargetRef: autoscaling.CrossVersionObjectReference{
 				Name: config.Name,
-				Kind: "DeploymentConfig",
+				Kind: "DeploymentConfig.deploy.openshift.io",
```

Comment:
I'm pretty sure that's the API version field that has the group and version squashed together with a slash :-(